### PR TITLE
[Page Shield] Add content

### PR DESCRIPTION
**Pull request by pedrosousa**
Addresses PCX-4605.

### CODE REVIEW

File: content/page-shield/_index.md [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
 weight: 1
+layout: single
 meta:
   title: Cloudflare Page Shield
 ---
 
 # Cloudflare Page Shield
 
-Page Shield provides you with visibility on the scripts loaded on your domain's pages, and helps you detect attacks happening in your end users' browsers.
+Page Shield provides you with visibility on your application's JavaScript dependencies and the destinations of the connections they make, and to receive alerts when they change. These features help you detect attacks happening in your end users' browsers.
```
Comment (marciocloudflare) [inline content/page-shield/_index.md]: The second part of the sentence sounds weird. Should it be "and allows you to receive alerts..."?

Comment (pedrosousa) [reply content/page-shield/_index.md]: Yes, sounds better!

File: content/page-shield/reference/script-statuses.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
 ---
 pcx_content_type: reference
-title: Script statuses
-weight: 1
+title: Statuses
+weight: 2
```
Comment (marciocloudflare) [inline content/page-shield/reference/script-statuses.md:4]: Consider adding a `meta:title:` as the full title for this page is `Script and connection statuses`

File: content/page-shield/reference/settings.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- No firewall rules match the `cdn-cgi/*` URL path
+
+### Configure the reporting endpoint
+
+To configure the CSP reporting endpoint:
+
+1.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account and domain.
+2.  Go to **Security** > **Page Shield** > **Settings**.
+3.  Under **Reporting endpoint**, select **Cloudflare-owned endpoint** or **Same hostname**.
+4.  Click **Apply settings**.
```
Comment (marciocloudflare) [inline content/page-shield/reference/settings.md]:
```suggestion
4.  Select **Apply settings**.
```

File: content/page-shield/reference/settings.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+When connection targets are reported to Cloudflare, their URIs can sometimes include sensitive data such as session ID.
+
+By default, Page Shield will only check the domain against malicious threat intelligence feeds. You can choose to let Page Shield use the full URI when analyzing the connections made from your domain's pages. Any sensitive data present in the URI will be logged in clear text, and any user with access to the connection monitor dashboard will be able to view it.
+
+### Configure the connection target details to use
+
+1.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account and domain.
+2.  Go to **Security** > **Page Shield** > **Settings**.
+3.  Under **Connection target details**, select **Log host only** to analyze only the hostname or **Log full URI** to use the full URI in Page Shield.
+4.  Click **Apply settings**.
```
Comment (marciocloudflare) [inline content/page-shield/reference/settings.md]:
```suggestion
4.  Select **Apply settings**.
```

File: content/page-shield/use-dashboard/monitor-connections.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+## Use the Active Connections dashboard
+
+To review the active connections identified by the connection monitor:
+
+1.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select your account and domain.
+2.  Go to **Security** > **Page Shield** > **Connection monitor**.
+3.  Under **Active connections**, review the active connections for your domain.
+4.  To filter connections, use the following options:
+
+    - **Search scripts**: Search for a specific URL.
```
Comment (marciocloudflare) [inline content/page-shield/use-dashboard/monitor-connections.md]: Just wanted to confirm this really searches for a specific URL, not script

**Review by marciocloudflare — APPROVED** [review]

File: content/page-shield/use-dashboard/monitor-connections.md (hunk 5 above)
Comment (pedrosousa) [reply content/page-shield/use-dashboard/monitor-connections.md]: Nice catch! It does search by URL, but it should be "connection", not "script".

**Pull request merged**